examples/fastify-ssr/src/app/AppShell.js

```diff
@@ -15,11 +14,10 @@ class AppShell extends React.Component {
           <Link to="/">Home</Link>
           <Link to="/users">PaginationPage</Link>
         </nav>
-        <Router>
```

Comment:
Router moved to parent component, so this one can be imported on server side, which uses a different kind of router.